airflow/operators/bash.py

```diff
@@ -100,8 +100,6 @@ def execute(self, context):
                                   for k, v in airflow_context_vars.items()]))
         env.update(airflow_context_vars)
 
-        self.lineage_data = self.bash_command
```

Comment:
Isn't it something related to lineage? 👀

Reply:
@ashb @mik-laj do you know if this is required?

Reply:
Yes it is required

Reply:
No, it is no longer required. The lineage backend has been modified and this was used earlier